lib/js/index.js

```diff
@@ -70,15 +72,17 @@ class DateTimePicker extends Events {
       styles: this.options.styles,
       time: false,
       dateValidator: validator,
-      initialValue: this.value
+      initialValue: this.options.default,
+      weekdayFormat: this.options.default.localeData().weekdaysMin(),
+      weekStart: this.options.default.clone().weekday(0).day()
```

Comment:
`WeekDayFormat` changes the names of the days displayed on the calendar according to the language.
`weekStart` sets the week start day according to the moment object received as parameter.